typescript/sdk/src/router/MultiProtocolRouterApps.ts

```diff
@@ -50,10 +49,15 @@ export class MultiProtocolGasRouterApp<
   ContractAddrs extends RouterAddress = RouterAddress,
   IAdapterApi extends IGasRouterAdapter = IGasRouterAdapter,
 > extends MultiProtocolRouterApp<ContractAddrs, IAdapterApi> {
-  public override readonly protocolToAdapter = {
-    [ProtocolType.Ethereum]: EvmGasRouterAdapter,
-    [ProtocolType.Sealevel]: SealevelGasRouterAdapter,
-  };
+  override protocolToAdapter(
```

Comment:
Noticing this is exactly the same as the super's `protocolToAdapter`, is it necessary?

Reply:
Good catch!